Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -317,4 +355,48 @@ export function select(element, elementId, focus) {
     if (picker && picker.altInput) {
         utilities.select(picker.altInput, null, focus);
     }
+}
+
+class ClassWatcher {
```

Comment:
We already have an observer. Can we make sure that the observer doesn't already do this. If it doesn't can we add this there?

Reply:
> Can we make sure that the observer doesn't already do this.

Not sure what you mean. Can you explain?

Reply:
![image](https://github.com/Megabit/Blazorise/assets/22283161/881a7773-c828-418a-a6fe-93a623916245)